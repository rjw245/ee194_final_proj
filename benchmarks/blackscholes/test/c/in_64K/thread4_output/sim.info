{'cmdline': ['/h/rwood04/ee194/sniper/run-sniper',
             '-d',
             './thread4_output',
             '-n',
             '8',
             '-c',
             'gainestown',
             '-c',
             'nehalem.cfg',
             '--',
             '../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-pthreads/bin/blackscholes',
             '4',
             '../../../inputs/in_64K.txt',
             'thread4_output/output.txt'],
 'git_revision': '',
 'host': 'dell26',
 'pin_version': 71313,
 'rusage': (12223.526741,
            90.601226,
            136500,
            0,
            0,
            0,
            222834,
            1,
            0,
            8,
            15000,
            0,
            0,
            0,
            14765032,
            6441006),
 'snipercmd': ['bash',
               '-c',
               '/h/rwood04/ee194/sniper/pin_kit/intel64/bin/pinbin -mt -injection child -xyzzy -enable_vsm 0 -t /h/rwood04/ee194/sniper/lib/pin_sim -c /h/rwood04/ee194/sniper/config/base.cfg --general/total_cores=8 --general/output_dir=/h/rwood04/ee194/ee194_docs/final_project/ee194_final_proj/benchmarks/blackscholes/test/c/in_64K/thread4_output --config=/h/rwood04/ee194/sniper/config/nehalem.cfg --config=/h/rwood04/ee194/sniper/config/gainestown.cfg --config=/h/rwood04/ee194/sniper/config/nehalem.cfg -g --traceinput/stop_with_first_app=true -g --traceinput/restart_apps=false -- ../../../parsec-3.0/pkgs/apps/blackscholes/inst/amd64-linux.gcc-pthreads/bin/blackscholes 4 ../../../inputs/in_64K.txt thread4_output/output.txt'],
 't_elapsed': 9298.131796836853,
 't_start': 1461613856.399475,
 'tracecmds': [],
 'user': 'rwood04',
 'vmem': 855113728L}
