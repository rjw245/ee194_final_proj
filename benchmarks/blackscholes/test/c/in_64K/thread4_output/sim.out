                                     | Core 0      | Core 1      | Core 2      | Core 3      | Core 4      | Core 5      | Core 6      | Core 7     
  Instructions                       |   745615679 |  1186316006 |  1186332906 |  1186330282 |  1186314644 |           0 |           0 |           0
  Cycles                             |  1274826280 |  1274826280 |  1274826280 |  1274826280 |  1274826280 |  1274826280 |  1274826280 |  1274826280
  IPC                                |        0.58 |        0.93 |        0.93 |        0.93 |        0.93 |        0.00 |        0.00 |        0.00
  Time (ns)                          |   479258000 |   479258000 |   479258000 |   479258000 |   479258000 |   479258000 |   479258000 |   479258000
  Idle time (ns)                     |   308398931 |   171401360 |   169290788 |   172384172 |   170862342 |   479258000 |   479258000 |   479258000
  Idle time (%)                      |       64.3% |       35.8% |       35.3% |       36.0% |       35.7% |      100.0% |      100.0% |      100.0%
Branch predictor stats               |             |             |             |             |             |             |             |            
  num correct                        |   118061534 |    69802422 |    69732539 |    69839393 |    69776179 |           0 |           0 |           0
  num incorrect                      |     6913412 |     2012757 |     2082540 |     1976056 |     2039001 |           0 |           0 |           0
  misprediction rate                 |       5.53% |       2.80% |       2.90% |       2.75% |       2.84% |       0.00% |       0.00% |       0.00%
  mpki                               |        9.27 |        1.70 |        1.76 |        1.67 |        1.72 |        0.00 |        0.00 |        0.00
TLB Summary                          |             |             |             |             |             |             |             |            
  I-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   122007124 |   133954374 |   133955574 |   133956364 |   133954707 |           0 |           0 |           0
    num misses                       |         107 |          23 |          23 |          23 |          23 |           0 |           0 |           0
    miss rate                        |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00 |        0.00
  D-TLB                              |             |             |             |             |             |             |             |            
    num accesses                     |   255523094 |   380354981 |   380358681 |   380358594 |   380354168 |           0 |           0 |           0
    num misses                       |      331368 |     6219155 |     6215056 |     6170480 |     6408654 |           0 |           0 |           0
    miss rate                        |       0.13% |       1.64% |       1.63% |       1.62% |       1.68% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.44 |        5.24 |        5.24 |        5.20 |        5.40 |        0.00 |        0.00 |        0.00
  L2 TLB                             |             |             |             |             |             |             |             |            
    num accesses                     |      331475 |     6219178 |     6215079 |     6170503 |     6408677 |           0 |           0 |           0
    num misses                       |        1951 |        3453 |       59904 |       12365 |       31356 |           0 |           0 |           0
    miss rate                        |       0.59% |       0.06% |       0.96% |       0.20% |       0.49% |       0.00% |       0.00% |       0.00%
    mpki                             |        0.00 |        0.00 |        0.05 |        0.01 |        0.03 |        0.00 |        0.00 |        0.00
Cache Summary                        |             |             |             |             |             |             |             |            
  Cache L1-I                         |             |             |             |             |             |             |             |            
    num cache accesses               |   122007124 |   133954374 |   133955574 |   133956364 |   133954707 |           0 |           0 |           0
    num cache misses                 |      250842 |        8838 |        8838 |        8881 |        8838 |           0 |           0 |           0
    miss rate                        |       0.21% |       0.01% |       0.01% |       0.01% |       0.01% |        inf% |        inf% |        inf%
    mpki                             |        0.34 |        0.01 |        0.01 |        0.01 |        0.01 |         inf |         inf |         inf
  Cache L1-D                         |             |             |             |             |             |             |             |            
    num cache accesses               |   255654297 |   380354987 |   380358687 |   380358602 |   380354173 |           0 |           0 |           0
    num cache misses                 |      194167 |     1119617 |     1094499 |     1105064 |     1122661 |           0 |           0 |           0
    miss rate                        |       0.08% |       0.29% |       0.29% |       0.29% |       0.30% |        inf% |        inf% |        inf%
    mpki                             |        0.26 |        0.94 |        0.92 |        0.93 |        0.95 |         inf |         inf |         inf
  Cache L2                           |             |             |             |             |             |             |             |            
    num cache accesses               |      445079 |     1128458 |     1103338 |     1113946 |     1131505 |           0 |           0 |           0
    num cache misses                 |      113516 |      731810 |      732018 |      731290 |      731256 |           0 |           0 |           0
    miss rate                        |      25.50% |      64.85% |      66.35% |      65.65% |      64.63% |        inf% |        inf% |        inf%
    mpki                             |        0.15 |        0.62 |        0.62 |        0.62 |        0.62 |         inf |         inf |         inf
  Cache L3                           |             |             |             |             |             |             |             |            
    num cache accesses               |      113594 |      731813 |      732019 |      731291 |      731262 |           0 |           0 |           0
    num cache misses                 |       68893 |        1170 |        1225 |        1161 |        7766 |           0 |           0 |           0
    miss rate                        |      60.65% |       0.16% |       0.17% |       0.16% |       1.06% |        inf% |        inf% |        inf%
    mpki                             |        0.09 |        0.00 |        0.00 |        0.00 |        0.01 |         inf |         inf |         inf
DRAM summary                         |             |             |             |             |             |             |             |            
  num dram accesses                  |       36180 |           0 |           0 |           0 |       36180 |           0 |           0 |           0
  average dram access latency (ns)   |       60.81 |         inf |         inf |         inf |       60.79 |         inf |         inf |         inf
  average dram queueing delay        |        7.38 |        0.00 |        0.00 |        0.00 |        7.36 |        0.00 |        0.00 |        0.00
  average dram bandwidth utilization |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00% |       0.00%
Coherency Traffic                    |             |             |             |             |             |             |             |            
  num loads from dram                |        8084 |        1083 |        1065 |        1088 |          59 |           0 |           0 |           0
  num loads from dram cache          |           0 |           0 |           0 |           0 |           0 |           0 |           0 |           0
  num loads from remote cache        |        1027 |          22 |           8 |          22 |        6432 |           0 |           0 |           0
